packages/react/webpack.mf.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const fs = require('fs');
+const webpack = require('webpack');
+const getResolveAlias = require('@fluentui/scripts/webpack/getResolveAlias');
+const { createServeConfig } = require('@fluentui/scripts/webpack/webpack-resources');
+const { webpackMerge } = require('just-scripts');
+
+const shared = {
```

Comment:
I know I say this all the time but it would probably help to have some comments in here in case anyone else needs to maintain it later.